lua/nvim-web-devicons/icons-default.lua

```diff
@@ -143,6 +143,18 @@ local icons_by_filename = {
     cterm_color = "113",
     name = "Zshrc",
   },
+  [".zsh_plugins.txt"] = {
+    icon = "󱉋",
+    color = "#ff0288",
+    cterm_color = "198",
+    name = "Antidote",
+  },
+  [".zsh_history"] = {
+    icon = "󰯂",
```

Comment:
Can we please use nf-seti-text `` ?

It's not been properly communicated however we are attempting to standardise on seti, with an alternate material theme.

Reply:
np ll fix

Reply:
can i use other seti icons on .zsh_history than nf-seti-text
im still will be using nf-seti though

Reply:
Some filetypes/files have the icons that cannot be substituted by nf-seti
such as archlinux or antidote

Reply:
> can i use other seti icons on .zsh_history than nf-seti-text such as 

Yes please, that would be great.

Reply:
> Some filetypes/files have the icons that cannot be substituted by nf-seti such as archlinux or antidote

No worries; best effort is good enough.

Reply:
I am not sure about couple of those, `.zsh_history` isn't standard zsh's history file, is it? Neither is `.zsh_plugins.txt`, right?

@alex-courtis Perhaps what we need is an easy way to alias icons by users?

Reply:
Plus one on that but you'll need to file a separate issue for this